packages/elements/src/internals/machines/sign-up/continue.machine.ts

```diff
@@ -62,6 +62,21 @@ export const SignUpContinueMachine = setup({
       context.parent.send({ type: 'NEXT', resource: (event as unknown as DoneActorEvent<SignUpResource>).output }),
     sendToLoading,
   },
+  guards: {
+    isMissingRequirements: ({ context }) =>
+      context.parent.getSnapshot().context.clerk?.client?.signUp?.status === 'missing_requirements',
+    requirementsMet: ({ context }) => {
```

Comment:
Per a convo with @octoper, we're going to rename this guard.

Reply:
Done!